single_pages/dashboard/store/orders.php

```diff
@@ -147,7 +147,7 @@
                 <tr>
                     <td><?= h($discount['odName']); ?></td>
                     <td><?= h($discount['odDisplay']); ?></td>
-                    <td><?= h($discount['odDeductFrom']); ?></td>
+                    <td><?= t(ucwords($discount['odDeductFrom'])); ?></td>
```

Comment:
I've used ucwords because the values of $discount array are lower case (as stored in db) and the translation item are with leading uppercase. the decision here is ( to my opinion ) a: leave it in that way or b: duplicate the entries in the po file. Up to you :)